### PR TITLE
AP_HAL: changes for final PocketPilot design

### PR DESCRIPTION
**Pull request by mirkix**
These are the final changes for the PocketPilot cape. The cape will fit on a [PocketBeagle Linux Board](http://beagleboard.org/pocket) and will be available soon (next weeks, hopefully). There will be an announcement on the ArduPilot Blog if it's available.

This is a project from @patrickpoirier51 and me.

### CODE REVIEW

File: libraries/AP_HAL_Linux/SPIDevice.cpp [hunk 1]
```diff
@@ -101,8 +101,8 @@ SPIDesc SPIDeviceManager::_device[] = {
 };
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_POCKET
 SPIDesc SPIDeviceManager::_device[] = {
-    SPIDesc("mpu9250",    1, 0, SPI_MODE_3, 8, SPI_CS_KERNEL,  1*MHZ, 11*MHZ),
-    SPIDesc("ms5611",     1, 1, SPI_MODE_3, 8, SPI_CS_KERNEL,  10*MHZ,10*MHZ),
+    SPIDesc("mpu9250",    1, 1, SPI_MODE_3, 8, SPI_CS_KERNEL,  1*MHZ, 11*MHZ),
+    SPIDesc("bmp280",     1, 0, SPI_MODE_0, 8, SPI_CS_KERNEL,  1*MHZ,10*MHZ),
```
Comment (tridge) [inline libraries/AP_HAL_Linux/SPIDevice.cpp]: I doubt the bmp280 needs different high and low bus speeds

Comment (patrickpoirier51) [reply libraries/AP_HAL_Linux/SPIDevice.cpp]: tested with 10 - 10  and its ok

**Comment by mirkix** [comment]
@tridge SPI speed of BMP280 is now set to 10MHz.

**Comment by patrickpoirier51** [comment]
Who is merging ?
I would like to test new release  from master

**Pull request merged**